google-cloud-pubsub/lib/google/cloud/pubsub/topic.rb

```diff
@@ -51,6 +52,33 @@ def initialize
           @grpc = Google::Pubsub::V1::Topic.new
           @name = nil
           @exists = nil
```

Comment:
Both `AsyncPublisher` and `BatchPublisher` have been moved from being nested under `Topic` to the top level.